scripts/build_plugins.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#
+#Copyright 2015 Intel Corporation
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__proj_dir="$(dirname "$__dir")"
+
+# shellcheck source=scripts/common.sh
+. "${__dir}/common.sh"
+
+_info "project path: ${__proj_dir}"
+
+git_version=$(_git_version)
+go_build=(go build -ldflags "-w -X main.gitversion=${git_version}")
+
+_info "git commit: $(git log --pretty=format:"%H" -1)"
+
+# Disable CGO for builds.
+export CGO_ENABLED=0
+
+# rebuild binaries:
+export GOOS=${GOOS:-$(uname -s | tr '[:upper:]' '[:lower:]')}
+export GOARCH=${GOARCH:-"amd64"}
+
+OS=$(uname -s)
+if [[ "${OS}" == "Darwin" ]]; then
+  p=$(type -p sysctl > /dev/null && sysctl -n hw.ncpu || echo "1")
```

Comment:
Is this possible to still have an option to manually define p? Some developer may prefer to wait longer but not have their laptop frozen.